torch/_dynamo/output_graph.py

```diff
@@ -595,9 +595,11 @@ def get_submodule(self, keys):
         return obj
 
     def new_var(self, name="tmp"):
+        graph_names = {node.name for node in self.graph.nodes}
```

Comment:
Naw man I still confused. Why do I care about conflicts with FX nodes when I am allocating names for co_varnames